.travis.yml

```diff
@@ -28,17 +31,27 @@ addons:
       - llvm-toolchain-precise
 
 env:
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Ninja"
-  - COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
-
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.7 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
-  - COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
-
-  - COMPILER=gcc CMAKE_GENERATOR="Ninja"
-  - COMPILER=gcc CMAKE_GENERATOR="Unix Makefiles"
+  # Test all generators and Clang + GCC
+  - TEST=true COMPILER=clang++-3.6 CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=clang++-3.6 CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=clang++-3.7 CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=clang++-3.7 CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=clang++-3.8 CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=g++-4.7     CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=g++-4.7     CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=g++-4.8     CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=g++-4.8     CMAKE_GENERATOR="Unix Makefiles"
+  - TEST=true COMPILER=g++-4.9     CMAKE_GENERATOR="Ninja"
+  - TEST=true COMPILER=g++-4.9     CMAKE_GENERATOR="Unix Makefiles"
```

Comment:
I don't think so, because these compiler versions technically do provide the features, but unfortunately it so happens that there's a bug in their implementation.